app/jobs/api/issues_label_issue_job.rb

```diff
@@ -1,8 +1,10 @@
 module Api
   class IssuesLabelIssueJob < IssueEventJob
-    include IsPublishable
```

Comment:
Didn't have time to implement slack and hipchat messages for label changes. 

Basically didn't find it necessary to publish the events any of the integrations.